src/.eslintrc.cjs

```diff
@@ -22,52 +23,54 @@ module.exports = {
         "quotes": [ "error", "double" ],
         "semi": [ "error", "always" ],
         "camelcase": "off",
-        // "no-unused-vars": "off",
         "eqeqeq": "off",
         "comma-dangle": [ "error", "never" ],
         "comma-style": [ "error", "last" ],
         "comma-spacing": "off",
```

Comment:
Remove this rule, please

Reply:
`"no-unused-vars": "off"` is removed

Reply:
notice: but we have it in `IMA` and in `network browser`

Reply:
removed `"comma-spacing": "off"`  rule